desc/protoparse/errors.go

```diff
@@ -92,15 +92,26 @@ type ErrorWithSourcePos struct {
 
 // Error implements the error interface
 func (e ErrorWithSourcePos) Error() string {
-	if e.Pos.Line <= 0 || e.Pos.Col <= 0 {
-		return fmt.Sprintf("%s: %v", e.Pos.Filename, e.Underlying)
+	filename := "<input>"
```

Comment:
This is just what I use in Buf, we could also make this "", or something else

Reply:
Let's push this down to `GetPosition` -- so that returns a `SourcePos` with `"<input>"` as the filename. Then this method can simply call `GetPosition` so it need not try to de-reference the pointer.

Reply:
Updated